### PR TITLE
Update build-configuration.md (add nx.dev)

### PR DESCRIPTION
**Pull request by waylun**

### CODE REVIEW

**Comment by kristianfreeman** [comment]
Hey @waylun - I see that the build directory you used includes the `apps` directory. I'm not familia with NX, but looking at the docs, it appears like that may be non-standard? Any thoughts there?

**Comment by waylun** [comment]
> Hey @waylun - I see that the build directory you used includes the `apps` directory. I'm not familia with NX, but looking at the docs, it appears like that may be non-standard? Any thoughts there?

Hi @signalnerve, thanks for the quick response. So NX is a monorepo that can host multiple applications in one repo, so the `apps` directory include as default for build distribution. NX.dev(Next.js)

File: products/pages/src/content/platform/build-configuration.md [hunk 1]
```diff
@@ -21,31 +21,32 @@ If you are not using a framework, you may input `exit 0` into the **Build comman
...
+| GitBook                      | `gitbook build`                                      | `_book`                         |
+| Gridsome                     | `gridsome build`                                     | `dist`                          |
+| Hugo                         | `hugo`                                               | `public`                        |
+| Jekyll                       | `jekyll build`                                       | `_site`                         |
+| Jigsaw                       | `vendor/bin/jigsaw build production`                 | `build_production`              |
+| Mkdocs                       | `mkdocs build`                                       | `site`                          |
+| Next.js (Static HTML Export) | `next build && next export`                          | `out`                           |
+| Nuxt.js                      | `nuxt generate`                                      | `dist`                          |
+| Nx.dev                       | `npx nx build $directory && npx nx export $directory`| `dist/apps/$directory/exported` |
```
Comment (lukeed) [inline products/pages/src/content/platform/build-configuration.md:41]: Everything else assumes that a project is already setup with the local/system bins are already in place.

```suggestion
| Nx.dev                       | `nx build $directory && nx export $directory`| `dist/apps/$directory/exported` |
```

**Comment by kristianfreeman** [comment]
Hey all - we've gotten some feedback internally from the team that NX isn't a good fit for this section, because it's a monorepo tool (not a framework). Going to close this!

**Pull request closed without merging**